templates/_components/quote_and_image.html.twig

```diff
@@ -1,35 +1,34 @@
-<div class="govuk-grid-row govuk-!-padding-9">
-    <div class="govuk-grid-column-one-quarter govuk-heading-xl govuk-!-padding-left-5 quote">“</div>
-    <div class="govuk-grid-column-one-third quote_image_width">
-        <div class="govuk-hint govuk-!-font-size-24">
-        {{ component.quote_and_image_quote_and_image_quote | striptags | raw }}
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">

```

Comment:
This doesn't seem right. Has this been done on other components? Looking at [page layout](
https://design-system.service.gov.uk/styles/layout/#page-wrappers) these classes are set on the page template, and we likely shouldn't repeat them on individual components

Reply:
checking over the components I have noticed they use a slightly different way to do the same thing on some components I will experiment to see if i can replicate the effect that this currently does

Reply:
I've now removed the main wrapper